<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QDialog" name="SecondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>518</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>隶书</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#SecondWindow{
background-color: rgb(99, 143, 172);
}
</string>
  </property>
  <widget class="QLabel" name="lblImage2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>240</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border:3px solid#63bdc5;
border-radius:200px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>258</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>学号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editId">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn2Confirm">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>确认录入</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>258</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editName">
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="videoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>                 视频流</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>240</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border:2px dotted#63bdc5;
border-radius:120px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2Backmain">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2Opencam">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
    </font>
   </property>
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2Shot">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>拍照</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>240</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/open.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2Change">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>400</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>修改数据库</string>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>lblImage2</zorder>
  <zorder>btn2Confirm</zorder>
  <zorder>videoLabel</zorder>
  <zorder>label</zorder>
  <zorder>btn2Backmain</zorder>
  <zorder>btn2Opencam</zorder>
  <zorder>btn2Shot</zorder>
  <zorder>btn2Change</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
